selfservice/flow/login/flow.go

```diff
@@ -47,6 +50,11 @@
 	ID  uuid.UUID `json:"id" faker:"-" db:"id" rw:"r"`
 	NID uuid.UUID `json:"-"  faker:"-" db:"nid"`
 
+	// HydraLoginChallenge is an optional field whose presence indicates that
+	// Kratos is being used as an identity provider in a Hydra OAUTH flow.
+	HydraLoginChallenge uuid.UUID           `json:"-" faker:"-" db:"hydra_login_challenge"`
```

Comment:
If you use `uuid.NullUUID` you can use a NULLable UUID. And please add a code doc for this value :)